operator/scheduler/server_test.go

```diff
@@ -178,103 +178,165 @@ func TestSubscribeServerEvents(t *testing.T) {
 	g := NewGomegaWithT(t)
 
 	type test struct {
-		name               string
-		existing_resources []client.Object
-		results            []*scheduler.ServerStatusResponse
+		name           string
+		existingServer mlopsv1alpha1.Server
+		response       *scheduler.ServerStatusResponse
 	}
 
 	// note expected state is derived in the test, maybe we should be explicit about it in the future
 	tests := []test{
 		{
-			// no scheduler state means lost servers metadata
-			name: "servers - no scheduler state",
-			existing_resources: []client.Object{
-				&mlopsv1alpha1.Server{
-					ObjectMeta: metav1.ObjectMeta{
-						Name:       "foo",
-						Namespace:  "default",
-						Generation: 1,
+			name: "server - with a valid scaling request",
```

Comment:
could we add a test case for `ServerStatusResponse_ScalingRequest` for a server that has no min and max replicas set in the CR?

Reply:
Yes, I'll add that.